static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -412,14 +412,14 @@ $pillars: (
         }
     }
 
-    background-color: $special-report-alt-faded !important;
+    background-color: $special-report-alt-faded;
 
     .fc-item__container.u-faux-block-link--hover {
-        background-color: $special-report-alt-faded !important;
+        background-color: darken($special-report-alt-faded, 2%);
 
         .fc-item__timestamp,
         .fc-trail__count--commentcount {
-            background-color: $special-report-alt-dark !important;
```

Comment:
`!important` is always a bad idea. In this particular case it was overriding the desired colour of the card (hover and non-hover state) when a SpecialReportAlt card was added to a container with a palette. Palette colours should take precedence.

![image](https://user-images.githubusercontent.com/19683595/226299697-869bd36e-aff1-4d8a-a40f-d2931d8543cf.png)

Reply:
🥳